kong.conf.default

```diff
@@ -234,7 +234,20 @@
                          # capabilities) pulling its configuration changes
                          # from the database.
                          #
-                         # Example: `stream_listen = 127.0.0.1:8444 http2 ssl`
+                         # Example: `admin_listen = 127.0.0.1:8444 http2 ssl`
+
+ #status_listen = off    # Comma-separated list of addresses and ports on
+                         # which the Status interface should listen.
+                         # The Status interface is a read-only endpoint
```

Comment:
I can't help but notice that the capitalized "Status" reads somewhat strangely. Beside that, I also can't help but notice that the terminology we use to refer to each listener interface is getting out of hand. I know this topic was debated internally, and I agree with "status" as a name. What I have an issue with is its strange capitalization and the qualifier ("interface").

As of this PR, we'd have:

- "proxy server"
- "Admin API"
- "stream mode" (also referred to as "stream proxy")
- "Status interface"

We should eventually address the naming inconsistencies among existing listeners, but we can make sure that this new listener uses a name we are comfortable with today, so that we avoid changing it later on.

I suggest "Status API" since it exposes a RESTful API just like the Admin API, while the proxy and stream listeners are reverse-proxy servers. If we were to stick with "interface", I'd suggest not capitalizing "status".

Reply:
"interface" was chosen to keep it consistent with "Admin interface".
I've changed it to "Status API".